File: HorseRacing.Contracts/Models/Authentication/Requests/LoginRequest.cs
using HorseRacing.Contracts.Base.Requests;
using HorseRacing.Contracts.Models.Authentication.Dtos;

namespace HorseRacing.Contracts.Models.Authentication.Requests
{
    /// <summary>
    /// Модель запроса для авторизации
    /// </summary>
    public class LoginRequest : BaseRequest<LoginDto>
    {
        public LoginRequest() : base() { }
        public LoginRequest(LoginDto data) : base(data) { }
    }
}

File: HorseRacing.Web/src/interfaces/api/contracts/model/auth/requests/login-request.ts
import type { BaseRequest } from "../../../base/requests/base-request";
import type { LoginDto } from "../dto/login-dto";

export class LoginRequest implements BaseRequest<LoginDto> {
    Data: LoginDto;

    constructor(data?: LoginDto) {
        this.Data = data || { Username: '', Password: '' };
    }
}